sormas-rest/src/main/java/de/symeda/sormas/rest/CaseResource.java

```diff
@@ -143,4 +145,21 @@ public Response updateExternalData(List<ExternalDataDto> externalData) {
 		}
 	}
 
+	/**
+	 * This endpoint is used to partially update the CaseData.
+	 * For allowing only a subset of the fields of the caseDataDto to be updated, and ensuring that the system can determine
+	 * which fields are not provided and which are intended to be reset it, the payload needs to be a Json object and not a dto
+	 *
+	 * @param uuid
+	 * @param caseDataDtoJson
+	 *            - a subset of caseDataDto fields, same structure as caseDataDto
+	 * @return - the updated caseDataDto
+	 * @throws Exception
+	 */
+	@POST
+	@Path("/postUpdate/{uuid}")
+	public CaseDataDto postUpdate(@PathParam("uuid") String uuid, JsonNode caseDataDtoJson) throws Exception {
```

Comment:
I'd like to add something like an "experimental" annotation to this, to make clear that this is for now only build for internal usage and still work in progress.
I searched for it, but haven't found something that would be a good fit. Maybe we create our own annotation?
@MateStrysewske Do you have an opinion on it?